### PR TITLE
feat(admin-ui): add support to update user role and remove user

### PR DESCRIPTION
**Pull request by rsbh**

### CODE REVIEW

File: ui/src/containers/organisations.list/users/columns.tsx [hunk 1]
```diff
@@ -30,19 +50,141 @@ export const getColumns: (
...
       footer: (props) => props.column.id,
     },
+    {
+      header: "Actions",
+      cell: ({ row, getValue }) => {
+        const userRoleIds = R.pipe(
+          R.pathOr([], [row?.original?.id || "", "roles"]),
+          R.map(R.path(["id"]))
+        )(userRolesMap);
+        const excluedRoles = R.filter(
```
Comment (anujk14) [inline ui/src/containers/organisations.list/users/columns.tsx:73]:
```suggestion
        const excludedRoles = R.filter(
```

File: ui/src/containers/organisations.list/users/index.tsx [hunk 2]
```diff
@@ -1,18 +1,58 @@
...
+        } = await client?.frontierServiceListOrganizationRoles(ordId, {
+          scopes: ["app/organization"],
+        });
+        const {
+          // @ts-ignore
+          data: { roles },
+        } = await client?.frontierServiceListRoles({
+          scopes: ["app/organization"],
+        });
+        setRoles([...roles, ...orgRoles]);
```
Comment (anujk14) [inline ui/src/containers/organisations.list/users/index.tsx:47]: Just confirming, this current PR adds functionality just to update org level roles, right?

Comment (rsbh) [reply ui/src/containers/organisations.list/users/index.tsx:47]: yes only org level roles

**Review by anujk14 — APPROVED** [review]
LGTM

**Pull request merged**